certificate-analytics-service/app/src/main/resources/logback/logback-spring-base.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is this necessary when we use spring boot structure logging? See hjmtj-import for reference.

Can for example just set this in application.properties: logging.structured.format.console=ecs